assets/tests/components/mapPage.test.tsx

```diff
@@ -148,10 +150,13 @@ jest.mock("../../src/hooks/useSearchResultsByCategory", () => ({
 
 jest.mock("../../src/helpers/fullStory")
 
-jest.mock("../../src/userInTestGroup")
-
 jest.mock("../../src/hooks/usePanelState")
 
+jest.mock("../../src/userTestGroups", () => ({
+  __esModule: true,
+  default: jest.fn(() => []),
+}))
```

Comment:
question/suggestion(non-blocking): does this work instead?
```suggestion
jest.mock("../../src/userTestGroups")
```

Reply:
It doesn't :(

Reply:
Can you show or explain what happens when it doesn't work?
My assumption is that this would break in the same way if we did https://app.asana.com/0/1148853526253420/1203482629607208/f

So my hypothesis is that we could do [the above](https://github.com/mbta/skate/pull/2377#discussion_r1492977637), and then  add
```
beforeEach(() => {
	jest.mocked(inTestGroup).mockReturnValue([])
})
```

Reply:
Alternatively, you can find if it breaks in the same way if you try
```
beforeEach(() => {
	jest.resetAllMocks()
})
```

Reply:
Oof! Sorry for my brevity on Friday! I meant to fill in more details, and I guess I wound up focused on other things 🤦 

The first suggestion - getting rid of the `__esModule: true, default: ...` leads to any test that interacts with test groups failing with 👇 this error 👇 

```
 TypeError: Cannot read properties of undefined (reading 'includes')

   10 |
   11 | const inTestGroup = (key: TestGroups): boolean => {
 > 12 |   return getTestGroups().includes(key)
      |                         ^
   13 | }
   14 |
   15 | export default inTestGroup
```

I think that's because without the default behavior given in the `__esModule: true, default: ...` block, `getTestGroups()` is returning `undefined` instead of `[]`, and then `userInTestGroup()` is blowing up.

The second suggestion resolves that by, as a `beforeEach`, restoring that default behavior, so the second suggestion does work properly. (I won't pretend to know exactly how `jest.mock(...)` and `jest.mocked(...)` play with each other, so I'm honestly not sure, but I'm guessing that putting this in a `beforeEach` rather than an initializer that just runs once when the test file loads is a better choice.)

The third suggestion, `jest.resetAllMocks()`, causes all tests in this file to fail with 👇 this error 👇 
```
TypeError: Cannot read properties of undefined (reading 'openVehiclePropertiesPanel')

  221 |   const defaultSearchOpen = !selectedEntity || !isMobile
  222 |
> 223 |   const { openVehiclePropertiesPanel } = usePanelStateFromStateDispatchContext()
      |                                                                                 ^
  224 |
  225 |   // #region Search Drawer Logic
  226 |   const [searchOpen, setSearchOpen] = useState<boolean>(defaultSearchOpen)

```

At a quick glance, I'm guessing that these tests just rely on a few mocks that aren't set up to accommodate those tests. I'd rather punt that setup to the Asana ticket you linked.